test/e2e/ginkgo/README.md

```diff
@@ -0,0 +1,25 @@
+# Kubestellar end to end testing
+The end to end testing includes:
+1. deployments are downsync propegated to the WECs.
+1. Update of the workload object on WDS should update the object on the WECs. Increase the number of replicas from 1 to 2, verify they are updated on the WECs.
+1. Changing the bindingpolicy objectSelector to no longer match should delete the object from the WECs.
+1. Changing the bindingpolicy objectSelector to match should create the object on the WECs.
+1. Delete of the bindingpolicy object should delete the object on the WECs.
+1. Delete of the overlapping bindingpolicy object should not delete the object on the WECs.
+1. Delete of the workload object on WDS deletes the relevant objects on the WECs.
+1. Delete of a bindingpolicy deletes the relevant objects on the WECs.
+1. Singleton status update
+1. Object field filtering for services
+1. Resiliency testing - killing kubestellar manager on the WDS
+1. Resiliency testing - killing kubeflex
+1. Resiliency testing - killing both kubestellar and kubeflex
+
+## Running the tests
+To install Ginkgo, follow the instructions in [Ginkgo's getting started](https://onsi.github.io/ginkgo/#getting-started).
+
+To execute these test units, issue `ginkgo -v`.  This will make a local image based and run the end to end tests.
+
+To test the latest release image, issue `ginkgo -v -- -released=on`
+
+To test a specific test unit use ginkgo's --focus parameter.  For example, `ginkgo -v --focus "survives IMBS vcluster coming down"`
```

Comment:
These are not "unit" tests. These are end-to-end tests.

Reply:
These are not unit tests.  These are test units, i.e. a single operation or set of operations that compose one test within a a test file. Is this term, test unit, no longer popular in the field?  If so I'll stop using it.

Reply:
Sorry, I misread. It is OK.